### PR TITLE
GL-381: List pseudo exemption api

### PR DESCRIPTION
**Pull request by rasikasri**
### Jira link

[GL-381](https://transformuk.atlassian.net/browse/GL-381)

### What?

I have added/removed/altered:

- [ ] Added admin api endpoint to list green lanes exemptions

### Why?

I am doing this because:

- User should be able to list exemptions in admin UI

### CODE REVIEW

File: spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    end
+
+    context 'with some exemption' do
+      before { exemption }
+
+      it { is_expected.to have_http_status :success }
+      it { expect(json_response).to include('data') }
+    end
+
+    context 'without any exemption' do
```
Comment (alexdesi) [inline spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb:23]: without any exemption**s**.

(any require plural)

File: spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  let(:json_response) { JSON.parse(page_response.body) }
+  let(:exemption) { create :green_lanes_exemption }
+
+  describe 'GET to #index' do
+    let(:make_request) do
+      authenticated_get api_admin_green_lanes_exemptions_path(format: :json)
+    end
+
+    context 'with some exemption' do
```
Comment (alexdesi) [inline spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb:16]: some require plural form: "with some exemption**s**"

File: spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      before { exemption }
+
+      it { is_expected.to have_http_status :success }
+      it { expect(json_response).to include('data') }
+    end
+
+    context 'without any exemption' do
+      it { is_expected.to have_http_status :success }
+      it { expect(json_response).to include('data' => []) }
```
Comment (alexdesi) [inline spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb:25]: Should we test the presence of the exception?

**Review by alexdesi — COMMENTED** [review]
it looks great,
just couple of minor comments in the specs

File: spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb (hunk 1 above)
Comment (rasikasri) [reply spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb]: done

File: spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb (hunk 2 above)
Comment (rasikasri) [reply spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb]: done

File: spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb (hunk 3 above)
Comment (rasikasri) [reply spec/requests/api/admin/green_lanes/exemptions_controller_spec.rb:26]: Added

**Review by jebw — APPROVED** [review]
LGTM

**Review by alexdesi — APPROVED** [review]
:white_check_mark:

**Pull request merged**